src/basics.md

```diff
@@ -1188,6 +1189,48 @@ Error level - description
 
 Run the recipe with `RUST_BACKTRACE=1` to display a detailed [`backtrace`] associated with this error.
 
+[ex-measure-elapsed-time]: #ex-measure-elapsed-time
+<a name="ex-measure-elapsed-time"></a>
+## Measure the elapsed time between two code sections
+
+[![std-badge]][std] [![cat-time-badge]][cat-time]
+
+One way to get the elapsed time is to directly use the method 
+[`time::Instant::elapsed`].
+
+```rust
+use std::time::{Duration, Instant};
+use std::thread;
+
+fn expensive_function() {
+    thread::sleep(Duration::from_millis(10));
+}
+
+fn main() {
+    let start = Instant::now();
+    expensive_function();
+    let duration = start.elapsed();
+    
+    println!("Time elapsed in expensive_function() is: {:?}", duration);
+}
+```
+Another way to get the time [`Duration`] is as a difference of two 
+time [`Instant`] objects.
+
+```rust,no_run
```

Comment:
I'd remove this second approach all together.

Reply:
Done